src/test/java/org/example/automation/amazon/DeliverToTest.java

```diff
@@ -11,9 +11,80 @@
 
 public class DeliverToTest extends BaseWebDriverTest {
 
-    @Test
-    public void verifyDeliverTo() {
-        webDriver.get("https://www.amazon.com");
+    public static final String AMAZON_COM = "https://www.amazon.com";
 
+    @Test(description = "The test checks shipping changes by US state zip codes.")
+    public void verifyDeliveryLocationChanging() {
+        webDriver.get(AMAZON_COM);
+
+        WebElement deliverIcon = webDriver.findElement(By.id("nav-global-location-data-modal-action"));
+        deliverIcon.click();
+
+        WebElement fieldUsZipCode = new WebDriverWait(webDriver, Duration.ofSeconds(20))
+                .until(ExpectedConditions.visibilityOfElementLocated(By.id("GLUXZipUpdateInput")));
+        fieldUsZipCode.sendKeys("95814");
+
+        WebElement buttonApply = webDriver.findElement(By.id("GLUXZipUpdate"));
+        buttonApply.click();
+
+        WebElement buttonContinue = new WebDriverWait(webDriver, Duration.ofSeconds(20))
+                .until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("div.a-popover-footer #GLUXConfirmClose")));
+        buttonContinue.click();
+
+        boolean isTrueDeliver = new WebDriverWait(webDriver, Duration.ofSeconds(20))
+                .until(ExpectedConditions.textToBe(By.id("glow-ingress-line2"), "Sacramento 95814\u200C"));

```

Comment:
Can I ask why\u200c is added? 
Thank you

Reply:
When we select delivery location with zip code 95814 of Sacramento , an invisible symbol added with code \u200C.